docs/workflow/building/libraries/README.md

```diff
@@ -259,8 +259,8 @@ dotnet.cmd pack src\libraries\System.Text.Json\src\ -c Release
 
 ## APICompat
 
-If changes to the library include any API incompatibilities, calling `dotnet build` (if the project isn't packable) or `dotnet pack` will result in API compatibility errors.
+If changes to the library include any API incompatibilities, calling `dotnet build` (if the project isn't packable) or `dotnet pack` (if the project is packable) will result in API compatibility errors.
```

Comment:
I don't think the (if the project isn't packable) and (if the project is packable) parts are necessary here.
```suggestion
If changes to the library include any API incompatibilities, calling `dotnet build` or `dotnet pack` may result in API compatibility errors.
```